frontend/src/console/components/NoticeBoard/NoticeBoard.tsx

```diff
@@ -198,12 +198,17 @@ const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
     <div className="explanation-container">
       <p>
         <WrappedMessage id="explanation" messages={messages} />
-        <br />
-        <br />
-        <WrappedMessage id="blogpost_text" messages={messages} />
-        <a href="https://opencraft.com/blog">
-          <WrappedMessage id="blogpost_link" messages={messages} />
-        </a>
+        <p />
+        {/* TODO: Point the link to a specific blog post, once it's created. */}
+        <WrappedMessage
+          id="blogpost_text"
+          messages={messages}
+          values={{
+            link: (text: string) => (
+              <a href="https://opencraft.com/blog">{text}</a>
```

Comment:
I forgot about the `target="_blank" rel="noopener noreferrer"` here. Please add it.

Reply:
I should've picked it up, too. Added